common/persistence/sql/class.Platform.php

```diff
@@ -151,7 +151,7 @@ public function getName(){
      * @return string
      */
     public function getNowExpression(){
-        $datetime = new DateTime();
+        $datetime = new DateTime('now', new \DateTimeZone('UTC'));
```

Comment:
There is a TIME_ZONE global constant defined in generis.conf.php. I think, it would be nice to use that here.

Reply:
times in database should always be UTC

Reply:
okay, I see. that global can be used for other reasons.